### DISCUSSION TITLE
Copyright statement in dashboard login page

### DISCUSSION THREAD

**Discussion by fsiddi** [mirumee/saleor-dashboard#2123] [General] [answered]
Currently the dashboard login page features an unclear copyright statement in the footer. In https://github.com/saleor/saleor-dashboard/pull/1703#discussion_r772221424 it was suggested to remove it, which makes sense. Is there a specific reason why it did not happen?

Having the statement removed would give more confidence to users/organizations that are using the software, regarding data ownership and identity.

**Comment by timuric** [comment, marked as answer]
@fsiddi sorry for the confusion, we will remove it

**Comment by orzechdev** [comment]
Removed in https://github.com/saleor/saleor-dashboard/pull/2139 and merged to the main branch. It should arrive in the next release.

**Reply by fsiddi** [reply]
Much appreciated!
